### PR TITLE
Final Milestone: Finalize "Hello World" RoR+React application.

### PR DESCRIPTION
**Pull request by SaidRasinlic**
### 🚀 In this milestone I've finalized my project following the next requirements :

- Create a new Rails app called 'hello-rails-react'.
- Use the commend rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
- Verify that the jsbundling-rails gem is added to your Gemfile.
- Initialize your project with Git.
- Install needed node dependencies as described in the previous lesson's tutorial.
- Create a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
- Update webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
- Create a static view that will be the root of your app.

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+// eslint-disable-next-line func-names
+module.exports = function (api) {
+  const validEnv = ['development', 'test', 'production'];
+  const currentEnv = api.env();
+  const isDevelopmentEnv = api.env('development');
+  const isProductionEnv = api.env('production');
+  const isTestEnv = api.env('test');
+
+  if (!validEnv.includes(currentEnv)) {
+    throw new Error(
```
Comment (Ismailco) [inline babel.config.js:10]: - [ ] Kindly in the babel.config.js, consider adding the following as in the guide
```javascript
module.exports = {
  presets: [
    [
      '@babel/preset-env',
      {
        targets: {
          node: 'current',
        },
      },
    ],
    '@babel/preset-react',
  ],
};
```

File: bin/rails [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby.exe
```
Comment (Ismailco) [inline bin/rails:1]: - [x] [OPTIONAL] Kindly remove the `.exe.` from this file so users using macOS or Linux can run your app, you can replace this string with `#!/usr/bin/env ruby`

File: bin/rake [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby.exe
```
Comment (Ismailco) [inline bin/rake:1]: - [x] [OPTIONAL] Kindly remove the `.exe.` from this file so users using macOS or Linux can run your app, you can replace this string with `#!/usr/bin/env ruby`

File: bin/setup [hunk 4]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ruby.exe
```
Comment (Ismailco) [inline bin/setup:1]: - [x] [OPTIONAL] Kindly remove the `.exe.` from this file so users using macOS or Linux can run your app, you can replace this string with `#!/usr/bin/env ruby`

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @SaidRasinlic, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: babel.config.js (hunk 1 above)
Comment (SaidRasinlic) [reply babel.config.js:10]: Hello, thank you for suggestions, however, **I've already added that part into my code** as you can see on the screenshot below, also due to compilation other code also was added automatically 😃 🥂. 

RoR-React-Required

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🍾🎉**
Hello @SaidRasinlic 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**